.travis.yml

```diff
@@ -24,20 +24,29 @@ before_install:
   - export PATH="$PATH:`pwd`/bin/"
   - ln -s `which chromium-browser` bin/google-chrome
 
-dart_task:
- - dartfmt
- - dartanalyzer
- # Split the tests into five shards to help parallelize them across Travis workers.
- - test: --preset travis --total-shards 5 --shard-index 0
- - test: --preset travis --total-shards 5 --shard-index 1
- - test: --preset travis --total-shards 5 --shard-index 2
- - test: --preset travis --total-shards 5 --shard-index 3
- - test: --preset travis --total-shards 5 --shard-index 4
+jobs:
+  include:
+    - stage: analyze_and_format
+      script: dartfmt_and_analyze.sh
```

Comment:
should we skip the shell script?

`dartfmt -n --set-exit-if-changed . && dartanalyzer .`